### ISSUE TITLE
Failed to multipart/form-data request with very long value

### ISSUE THREAD

**Issue by care0717** [k1LoW/runn#694]
I'm trying to upload a file to a GraphQL endpoint like as follows.
```
runners:
  req:
    endpoint: https://example.com
steps:
  update:
    req:
      /graphql:
        post:
          body:
            multipart/form-data:
              - operations: '{"query":"mutation veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongMutation($file:Upload!){hoge(file:$file){fuga{id}}}","variables":{"file":{}}}'
              - map: '{ "0": ["variables.file"] }'
              - 0: ./example.csv
```
But this request fails with below error.
```
operator.go:1435: http request failed on "[No Description]".steps.update: stat mutation/{"query":"mutation veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongMutation($file:Upload!){hoge(file:$file){fuga{id}}}","variables":{"file":{}}}: file name too long
```

I think this error is occurring https://github.com/k1LoW/runn/blob/0f7dec1ae5f1c43dd24fbd4c6a9aeba031da6bf1/http.go#L241, but I don't know how to solve this issue as there is no good way to distinguish between files and non-files in advance.
Is it better to handle `file name too long` ad hoc?. Let me know what you think. I would like to contribute.

**Comment by k1LoW** [comment]
```yaml
      /graphql:
        post:
          body:
            multipart/form-data:
              - operations: '{"query":"mutation veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongMutation($file:Upload!){hoge(file:$file){fuga{id}}}","variables":{"file":{}}}'
              - map: '{ "0": ["variables.file"] }'
              - 0: ./example.csv
```

`multipart/form-data:` should consist of a map, not a list.
Perhaps that is what is being affected.

https://github.com/k1LoW/runn/blob/0f7dec1ae5f1c43dd24fbd4c6a9aeba031da6bf1/testdata/book/http_multipart.yml#L9-L34

**Comment by care0717** [comment]
> multipart/form-data: should consist of a map, not a list.

`multipart/form-data` also works with list. https://github.com/k1LoW/runn/blob/0f7dec1ae5f1c43dd24fbd4c6a9aeba031da6bf1/http_test.go#L209-L222
The code in this issue will actually work if I shorten the mutation query.

Also, even if I use map, I get the same filename too long error.

**Comment by k1LoW** [comment]
> multipart/form-data also works with list.

Oh. I was incorrect.

**Comment by k1LoW** [comment]
@care0717 
Fixed version has been released as v0.90.4. Please try it out.

**Comment by care0717** [comment]
I confirmed that the error has been resolved.
Thanks for your quick fix!
